svcomp-benchmark/scripts/README.md

```diff
@@ -0,0 +1,13 @@
+#### Requirements
+* python-2.7
+
+#### Running the Scripts
+
+1. Install Python Dependencies
+	* `pip install subprocess32`
+
+2. Run `python svcomp-runner.py <SV-Comp benchmarks path> <Log file path>`. The results of running the RV-Match on the benchmarks are stored at the location specified by the `log file path` argument to the python process. So, for instance, if the sv-benchmarks reside at "/home/rvmatch/sv-benchmarks/c", and I want the result files to be stored at "/home/rvmatch/results", the call to the python script would look like `python /home/rvmatch/sv-benchmarks/c /home/rvmatch/results`.
+
```

Comment:
The call to the script above does not look right; don't you need to mention a `.py` file?